backend/src/service/generic.ts

```diff
@@ -82,8 +84,53 @@ export function genericService<T>(
     }
   };
 
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  const get = async (id: string) => {
+    // console.log(tableName);
+    let query = `SELECT * from ${tableName} WHERE id=${id} `;
+    // console.log(query);
+    const result = await connection.query(
+      // "SELECT * FROM authors WHERE id = $1;",
+      // [authorId]
+      query
+    );
+
+    if (result.rows.length == 0) {
+      throw new NotFoundError("Not found.");
+    } else {
+      const got = new ModelClass(result.rows[0]);
+      return got;
+    }
+  };
```

Comment:
Nice and clean! Great code!
I'd just cleanup the comments before merging, if all is working :)